src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -236,7 +241,8 @@ const ChatItemMyTextV2 = ({
     <View style={styles.chatContainer}>
       <View style={handleTextContainerStyle()}>
         {!isContinuous && (
-          <View style={styles.chatTitleContainer}>
+          <View
+            style={[styles.chatTitleContainer, attachments.length > 0 ? {marginBottom: 4} : {}]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style for `chatTitleContainer` has been updated to include extra margin when attachments are present. This is a good approach as it ensures that the UI adapts to different content types (text vs attachments).
```diff
- <View style={styles.chatTitleContainer}>
+ <View style={[styles.chatTitleContainer, attachments.length > 0 ? {marginBottom: 4} : {}]}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->